src/ServiceControl.Audit/Infrastructure/Hosting/HostArguments.cs

```diff
@@ -149,7 +149,7 @@ public void PrintUsage()
             using (
                 var stream =
                     Assembly.GetCallingAssembly()
-                        .GetManifestResourceStream("ServiceControl.Hosting.Help.txt"))
+                        .GetManifestResourceStream("ServiceControl.Audit.Infrastructure.Hosting.Help.txt"))
             {
                 if (stream != null)
```

Comment:
I think this should be an assert or an invalidoperationexception? It is expected that it is not null